examples/custom_shaders/CMakeLists.txt

```diff
@@ -36,3 +36,8 @@ target_link_libraries(custom_shaders
   ${GLEW_LIBRARIES}
   ${IGNITION-RENDERING_LIBRARIES}
 )
+
+add_custom_command(TARGET custom_shaders POST_BUILD
+  COMMAND ${CMAKE_COMMAND} -E copy_directory
+  ${CMAKE_SOURCE_DIR}/media
```

Comment:
Maybe you forgot to commit this directory?

```
Error copying directory from "/home/chapulina/dome_ws/src/ign-rendering/examples/custom_shaders/media" to "/home/chapulina/dome_ws/src/ign-rendering/examples/custom_shaders/build/media".
```

Reply:
oops, added in f36a6134cd0d01a63b8d819b783434eb8c8c8f0a